stdlib/universe/schema_functions_test.go

```diff
@@ -1474,19 +1499,19 @@ func TestDropRenameKeep_Process(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			t.Skip("https://github.com/influxdata/flux/issues/2490")
-			executetest.ProcessTestHelper(
+			// t.Skip("https://github.com/influxdata/flux/issues/2490")
+			executetest.ProcessTestHelper2(
 				t,
 				tc.data,
 				tc.want,
 				tc.wantErr,
-				func(d execute.Dataset, c execute.TableBuilderCache) execute.Transformation {
-					ctx := dependenciestest.Default().Inject(context.Background())
-					tr, err := universe.NewSchemaMutationTransformation(ctx, tc.spec, d, c)
+				func(id execute.DatasetID, mem *memory.Allocator) (execute.Transformation, execute.Dataset) {
+					spec := tc.spec.(*universe.SchemaMutationProcedureSpec)
+					tr, d, err := universe.NewSchemaMutationTransformation(context.Background(), spec, id, mem)
```

Comment:
I guess that whether we use the deprecated transformations will depend on the value of a feature flag defined in `influxdb`?

Do we want to exercise the deprecated and new versions of the transformation here, so that we can be confident that we'll still get correct behavior if we need to flip the feature off?